backend/Exceptions/Path/PathException.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Internships\Exceptions\Path;
+
+use Internships\Exceptions\SimpleMessageException;
+use Throwable;
+
+class PathException extends SimpleMessageException
+{
+    protected string $fullPath;
+
+    public function __construct(string $fullPath, int $code = 0, Throwable $previous = null)
+    {
+        $this->fullPath = $fullPath;
+        parent::__construct($code, $previous);
+    }
+
+    protected function newExceptionMessage(): string
+    {
+        return "There was trouble processing path {$this->fullPath}.";
+    }
+
+    protected function getStatusCode(): int
+    {
+        return static::INTERNAL_SERVER_ERROR;
```

Comment:
You will change this response code.
If u return code 500, u said:
- overloading the server with the number of queries,
- script error,
- error in the .htaccess file,
- incorrect file access level setting.